src/writer.jl

```diff
@@ -161,11 +161,26 @@ function write(path::AbstractString, obj; force=false)
 
     # Close .shp file
     close(io)
-    
+
     # Write .shx file
     index_handle = IndexHandle(header, shx_indices)
     Base.write(paths.shx, index_handle)
 
+    # Write .prj file
+    crs = try
+        GI.crs(obj)
+    catch
+        nothing
+    end
+    if !isnothing(crs)
+        try
+            Base.write(paths.prj, convert(GeoFormatTypes.ESRIWellKnownText, crs).val)
+        catch
+            @warn ".prj write failure: Could not convert CRS of type `$(typeof(crs))` to " *
+            "`GeoFormatTypes.ESRIWellKnownText`.  `using ArchGDAL` may load the necessary " *
+            "`Base.convert` method."
+        end
+    end
```

Comment:
How's this now?

Reply:
Looks good!

Reply:
This will always yield the following warning, because `ESRIWellKnownText` actually instantiates to `ESRIWellKnownText{GeoFormatTypes.Unknown}`. It should be `GeoFormatTypes.CRS()` here. We should also change https://github.com/JuliaGeo/Shapefile.jl/blob/main/src/handle.jl#L57 to include `GeoFormatTypes.CRS()` as first argument.

```
┌ Warning: .prj write failure: Could not convert CRS of type `ESRIWellKnownText{GeoFormatTypes.Unknown}` to `GeoFormatTypes.ESRIWellKnownText`.  `using ArchGDAL` may load the necessary `Base.convert` method.
```

Reply:
Might be good to check that our tests include reading and writing the same shapefile completely, so we at least hit this codepath once.

Reply:
We can't test the roundtrip write-then-read until we can write .dbf, but I made the change to use CRS and added a test for warning before `using ArchGDAL` and no warning after.